src/audio/pipeline/pipeline-graph.c

```diff
@@ -207,7 +209,8 @@ int pipeline_free(struct pipeline *p)
 		rfree(p->pipe_task);
 	}
 
-	ipc_msg_free(p->msg);
+	if (p->msg)
+		ipc_msg_free(p->msg);
```

Comment:
I think this is not needed. ipc_msg_free() checks for a NULL input value.

Reply:
sure, thanks